src/main/java/com/commercetools/sync/inventories/SyncProblemException.java

```diff
@@ -0,0 +1,10 @@
+package com.commercetools.sync.inventories;
+
+/**
+ * Indicates whether sync meets problem which makes further sync unavailable. It may refer to different scope of data,
+ * that are unable to be synced.
+ * e.g. May occur during fetching supply channels what makes all drafts unable to be synced.
+ * e.g.2. May occur during fetching batch of old inventory entries what makes batch of draft unable to be synced.
+ */
+class SyncProblemException extends RuntimeException {
```

Comment:
I don't really see any purpose of that if library provides error callback for client
which can act accordingly
plus by looking at the usages of this exception I can tell that it will never leave the scope of any
completion stage - it is swallowed inside other .exceptionally() handlers
regardin name of this class:
every exception indicates a Problem. Don't provide it in the name